controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1123,6 +1163,67 @@ int cli_bml::bml_rdkb_steering_client_measure_caller(int numOfArgs)
 }
 #endif //BEEROCKS_RDKB
 
+int cli_bml::set_dcs_continuous_scan_enable_caller(int numOfArgs)
+{
+    if (numOfArgs == 2) {
+        return set_dcs_continuous_scan_enable(args.stringArgs[0], args.intArgs[1]);
+    }
+    return -1;
+}
+
+int cli_bml::get_dcs_continuous_scan_enable_caller(int numOfArgs)
+{
+    if (numOfArgs == 1) {
```

Comment:
I've been using Yoda conditions for a long time now that these lines make me feel uncomfortable. 
Some static code analyzers would also complain.

Reply:
I am familiar with this approach - but this is not maintained in the prpl.

if @tomereli and @arnout  want to add this requirement to prplmesh guidelines it is OK, but a short search of the following regex. <[a-z,A-Z] == [0-9]> produced 375 results in 93 files

will not be handled in this PR, but we can set a task for this change across the code if decided

Reply:
Fix this, we will.
![image](https://user-images.githubusercontent.com/13386849/72826517-ab438c80-3c81-11ea-8b3e-4b94d744280f.png)